neuralop/models/fno_block.py

```diff
@@ -1,10 +1,39 @@
 from torch import nn
 import torch.nn.functional as F
-
+import torch
 from .spectral_convolution import FactorizedSpectralConv
 from .skip_connections import skip_connection
 from .mlp import MLP
 
+def resample(x, res_scale, axis):
+    if isinstance(axis, list) and isinstance(res_scale, (float, int)):
```

Comment:
Could be a tuple -- you'd want to check if it's iterable instead. 
Here, let's just check that they are both int or not:
```python
if isinstance(res_scale, (float, int)):
   if isinstance(axis, (float, int)):
       axis = [axis]
       res_scale = [res_scale]
    else:
        res_scale = [res_scale]*len(axis)
else:
    assert len(res_scale) == len(axis), error_msg
```